dashboard/app/views/shared/_course_tall_block.haml

```diff
@@ -35,13 +37,14 @@
 
 - if %w(course1 course2 course3 course4).include?(id)
   - audience = data_t_suffix('script.name', id, 'description_audience')
-  - block_class = 'courseblock-tall courseblock-tall-audiencerow'
-- else
-  - block_class = 'courseblock-tall'
+  
+- block_class = 'courseblock-tall'
+
+- if ! (current_user && showState)
+  - block_class << 'courseblock-noaction'
```

Comment:
did this work?  i think it needed a space inside the single quotes like this: ' courseblock-noaction'